src/libsync/configfile.cpp

```diff
@@ -258,6 +259,12 @@ int ConfigFile::timeout() const
     return settings.value(QLatin1String(timeoutC), 300).toInt(); // default to 5 min
 }
 
+bool ConfigFile::overrideServerChunkSize() const
+{
+    QSettings settings(configFile(), QSettings::IniFormat);
+    return settings.value(QLatin1String(overrideServerChunkSize), false).toBool();
```

Comment:
the constant should be renamed to `overrideServerChunkSizeC`, otherwise this very method will be referenced here

Reply:
Yikes, what a thing to miss, thanks